dev/com.ibm.ws.security.oidc.client_fat.2/fat/src/com/ibm/ws/security/openidconnect/client/fat/IBM/OidcClientWasReqURLTests.java

```diff
@@ -796,9 +796,13 @@ public void testWasReqURLOidc_cookie(String updatedHost, String rpServerConfig,
                 break;
             case EXCEPTION:
                 expectations = vData.addSuccessStatusCodesForActions(expectations, Constants.LOGIN_USER, Constants.GOOD_OIDC_LOGIN_ACTIONS_SKIP_CONSENT);
-                expectations = vData.addExpectation(expectations, Constants.LOGIN_USER, Constants.EXCEPTION_MESSAGE, Constants.STRING_CONTAINS, "Did NOT get expected exception",
-                                                    null, "timed out");
-                break;
+		// have been getting an exception with "timed out" in it, now we have a jdk that is returning "timeout"
+		// the exception checking code can't handle matches (to use ".*time.*out.*") so, we'll use 2 checks
```

Comment:
Can the exception checking code be updated to look for regexes?

Reply:
I was going to open an issue - just don't want to mess with it right now as I'm buried in other things.

Reply:
Just opened https://github.com/OpenLiberty/open-liberty/issues/20678.
It shouldn't take much time, but, every time I say that, ...